### PR TITLE
Combined profile_details and HomeScreenActivity and added coverUrl to tagged places

### PR DESCRIPTION
**Pull request by alveerak**
Sorry this was such a big pull request, I was combining the fragment and activity.

### CODE REVIEW

File: app/src/main/res/layout-land/activity_home_screen.xml [hunk 1]
```diff
@@ -7,31 +7,82 @@
...
+        android:layout_marginTop="23dp"
+        android:src="@drawable/com_facebook_profile_picture_blank_square" />
+
+    <TextView
+        android:id="@+id/tv_user_name"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignEnd="@+id/add_likes_btn"
+        android:layout_alignParentTop="true"
+        android:layout_marginTop="40dp"
```
Comment (samhithak) [inline app/src/main/res/layout-land/activity_home_screen.xml]: Move to dimens.xml

File: app/src/main/res/layout-land/activity_home_screen.xml [hunk 2]
```diff
@@ -7,31 +7,82 @@
...
+        android:text="@string/your_name"
+        android:textSize="@dimen/name_text_size" />
+
+    <TextView
+        android:id="@+id/tv_user_hometown"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignEnd="@+id/add_likes_btn"
+        android:layout_alignParentTop="true"
+        android:layout_marginTop="91dp"
```
Comment (samhithak) [inline app/src/main/res/layout-land/activity_home_screen.xml]: Same as above

File: app/src/main/res/layout-land/activity_home_screen.xml [hunk 3]
```diff
@@ -7,31 +7,82 @@
...
+        android:text="@string/your_hometown" />
+
+    <TextView
+        android:id="@+id/tv_games_played"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignEnd="@+id/add_likes_btn"
+        android:layout_alignParentTop="true"
+        android:layout_below="@+id/tv_user_hometown"
+        android:layout_marginTop="130dp"
```
Comment (samhithak) [inline app/src/main/res/layout-land/activity_home_screen.xml]: Same as above

File: app/src/main/res/layout-land/activity_home_screen.xml [hunk 4]
```diff
@@ -7,31 +7,82 @@
...
+        android:text="@string/num_games" />
 
     <Button
         android:id="@+id/play_game_btn"
-        android:layout_width="@dimen/std_btn_width"
-        android:layout_height="@dimen/std_btn_height"
-        android:layout_centerInParent="true"
+        android:layout_width="200dp"
```
Comment (samhithak) [inline app/src/main/res/layout-land/activity_home_screen.xml]: here as well

File: app/src/main/res/layout-land/activity_home_screen.xml [hunk 5]
```diff
@@ -7,31 +7,82 @@
...
+        android:layout_height="60dp"
+        android:layout_alignParentStart="true"
+        android:layout_alignTop="@+id/add_likes_btn"
+        android:layout_marginStart="92dp"
```
Comment (samhithak) [inline app/src/main/res/layout-land/activity_home_screen.xml]: Same

File: app/src/main/res/layout-land/activity_home_screen.xml [hunk 6]
```diff
@@ -7,31 +7,82 @@
...
+        android:layout_marginTop="130dp"
+        android:text="@string/num_games" />
 
     <Button
         android:id="@+id/play_game_btn"
-        android:layout_width="@dimen/std_btn_width"
-        android:layout_height="@dimen/std_btn_height"
-        android:layout_centerInParent="true"
+        android:layout_width="200dp"
+        android:layout_height="60dp"
```
Comment (samhithak) [inline app/src/main/res/layout-land/activity_home_screen.xml]: same

File: app/src/main/res/layout-land/activity_home_screen.xml [hunk 7]
```diff
@@ -7,31 +7,82 @@
...
-        android:layout_height="@dimen/std_btn_height"
+    <TextView
+        android:id="@+id/logout_txt"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
         android:layout_alignParentBottom="true"
-        android:layout_centerHorizontal="true"
-        android:layout_marginBottom="@dimen/buttons_vertical_padding"
+        android:layout_alignParentEnd="true"
+        android:padding="20dp"
```
Comment (samhithak) [inline app/src/main/res/layout-land/activity_home_screen.xml]: same

File: app/src/main/res/layout-land/activity_home_screen.xml [hunk 8]
```diff
@@ -7,31 +7,82 @@
...
         android:paddingHorizontal="@dimen/buttons_vertical_padding"
-        android:text="@string/logout_btn" />
+        android:text="@string/logout_btn"
+        android:textColor="@color/convo_blue"
+        android:textSize="20sp" />
```
Comment (samhithak) [inline app/src/main/res/layout-land/activity_home_screen.xml]: same

File: app/src/main/res/layout-land/activity_home_screen.xml [hunk 9]
```diff
@@ -7,31 +7,82 @@
...
+
+    <TextView
+        android:id="@+id/tv_three_likes"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentEnd="true"
+        android:layout_alignParentTop="true"
+        android:layout_marginEnd="93dp"
```
Comment (samhithak) [inline app/src/main/res/layout-land/activity_home_screen.xml]: same

File: app/src/main/res/layout-land/activity_home_screen.xml [hunk 10]
```diff
@@ -7,31 +7,82 @@
...
+        android:layout_marginTop="169dp" />
```
Comment (samhithak) [inline app/src/main/res/layout-land/activity_home_screen.xml]: here

File: app/src/main/java/a3/com/convo/activities/HomeScreenActivity.java [hunk 11]
```diff
@@ -51,5 +58,109 @@ public void onClick(View view) {
...
+            return;
+        }
+        else {
+            final String[] likesSentence = {"Some of the pages you like on facebook include...\n"};
+            for (int i = 0; i < 3 && i < userLikes.size(); i++) {
+                String likeObjectId = userLikes.get(i);
+                if (likeObjectId == null || likeObjectId.isEmpty()) {
+                    continue;
+                }
+                ParseQuery<Page> query = ParseQuery.getQuery(Page.class);
```
Comment (samhithak) [inline app/src/main/java/a3/com/convo/activities/HomeScreenActivity.java]: Move this to a separate method

Comment (alveerak) [reply app/src/main/java/a3/com/convo/activities/HomeScreenActivity.java]: Oops sorry I think I missed the file!

Comment (alveerak) [reply app/src/main/java/a3/com/convo/activities/HomeScreenActivity.java]: I’ll just put all of the setting UI contents in separate functions so it’s cleaner.

File: app/src/main/res/layout-land/activity_home_screen.xml (hunk 1 above)
Comment (alveerak) [reply app/src/main/res/layout-land/activity_home_screen.xml]: Oops I think I missed the entire landscape file! Thanks!

File: app/src/main/java/a3/com/convo/activities/HomeScreenActivity.java [hunk 12]
```diff
@@ -51,5 +63,125 @@ public void onClick(View view) {
...
+            }
+        });
+
+        ParseUser user = ParseUser.getCurrentUser();
+        if (user == null) {
+            Log.e("ProfileDetailsFragment", "Somehow user was logged out of parse?");
+            return;
+        }
+
+        set_user_hometown(user);
```
Comment (samhithak) [inline app/src/main/java/a3/com/convo/activities/HomeScreenActivity.java]: rename all these methods in camelCase setUserHometown

**Pull request merged**